src/lib/plugins/links.ts

```diff
@@ -312,6 +302,70 @@ function handleNewURLs(
 
 function replaceInternalLinks(href: string, base: string) {
   let newHref = href;
+  if(newHref.startsWith('https://github.com/FuelLabs/sway-standards/') && newHref.endsWith('.md') && !newHref.includes('README.md')){
+    const split = newHref.split('/');
+    let docName = split[split.length -1].replace(".md", "");
+    switch (docName) {
+      case 'src-2':
+        docName = 'src-2-inline-documentation'
```

Comment:
why do we need to change the doc name?

Reply:
I just changed the doc names in the source repo, which broke the old links. This patches the links until they get fixed in the other source repos